### PR TITLE
Allow setting of airbrake host to allow use with errbit

### PR DESCRIPTION
**Pull request by jrhe**

### CODE REVIEW

**Comment by mixonic** [comment]
@jrhe can you clarify what is happening here? Errbit uses the same API as airbrake?

File: index.js [hunk 1]
```diff
@@ -3,7 +3,13 @@
 
 module.exports = {
   name: 'ember-cli-airbrake',
+  isEnabled: function () {
```
Comment (mixonic) [inline index.js]: `isConfigured` seems more appropriate.

**Comment by jrhe** [comment]
Errbit uses the same API as airbrake yes but it will be on your own host/ endpoint rather than the default one. For this reason you need to set the host. This could be done afterwards or in a custom initializer and ordered after the initializer for ember-cli-airbrake to ensure its configured asap before errors are generated but its nicer putting it in the addon.

File: index.js (hunk 1 above)
Comment (jrhe) [inline index.js]: This isn't checking the configuration in config/environment but instead the value passed to EmberApp in the users Brocfile when they build. This allows the user to either enable or disable it independently of it being configured.

N.b, this should be a separate PR really. Been messy and haven't worked on a branch. Can break up if you want.

Comment (mixonic) [inline index.js]: Why not just use the presence of a config value in the `config/environment.js` as the toggle? I'm not sure two flags are warranted when they will change pretty much in unison.

**Comment by mixonic** [comment]
Would love to see some basic documentation and a unified config. For reference we read the config at build-time in Torii:
- https://github.com/Vestorly/torii/blob/master/ember-addon/index.js#L26

Excellent addition!
